src/components/HotelsList.jsx

```diff
@@ -0,0 +1,148 @@
+import React, { useEffect, useState } from "react";
+import { connect } from "react-redux";
+import { getHotels } from "../modules/destination.js";
+import { Map, InfoWindow, GoogleApiWrapper, Marker } from "google-maps-react";
+import { Button } from "semantic-ui-react";
+import { chooseHotel } from "../modules/destination.js";
+
+const HotelsList = props => {
+  const [gotHotelsData, setGotHotelsData] = useState(false);
+  const [showingInfoWindow, setShowingInfoWindow] = useState(false);
+  const [activeMarker, setActiveMarker] = useState({});
+  const [selectedPlace, setSelectedPlaces] = useState({});
+  const [hotelMessage, setHotelMessage] = useState("");
+
+  const getHotelsShowData = async () => {
+    const hotelsData = await getHotels(
```

Comment:
suggest to keep it consistent with just response. also less confusion with similar variablenames

Reply:
changed